projects/plugins/social/src/class-meta-tags.php

```diff
@@ -342,8 +342,33 @@ public function render_tags( $post = null ) {
 			);
 		}
 
-		// Add SIG image if enabled.
-		$tags = apply_filters( 'jetpack_open_graph_tags', $tags );
+		/**
+		 * Filter the minimum width of the images used in Jetpack Open Graph Meta Tags.
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param int 200 Minimum image width used in Jetpack Open Graph Meta Tags.
+		 */
+		$image_width = absint( apply_filters( 'jetpack_open_graph_image_width', 200 ) );
+
+		/**
+		 * Filter the minimum height of the images used in Jetpack Open Graph Meta Tags.
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param int 200 Minimum image height used in Jetpack Open Graph Meta Tags.
+		 */
+		$image_height = absint( apply_filters( 'jetpack_open_graph_image_height', 200 ) );
+
+		/**
+		 * Allow the addition of additional Open Graph Meta tags, or modify the existing tags.
+		 *
+		 * @since 1.9.0
+		 *
+		 * @param array $tags Array of Open Graph Meta tags.
+		 * @param array $args Array of image size parameters.
```

Comment:
This is an odd description of the parameter isn't it? It's actually the minimum size of the images we want to include in the OG tags.